### DISCUSSION TITLE
I am not able to add my repository to the Community Exchange

### DISCUSSION THREAD

**Discussion by ArtyomVancyan** [community/community#46346] [GitHub Education] [answered]
On the [Community Exchange](https://education.github.com/globalcampus/exchange) page, I was trying to add my repository, but I am not able to see my public repository in the list because it is forked from my organization and does not have an issues section. I would like to add the original one (not the forked one) if it was possible. How can I add a repository of my own organization?

**Comment by jge162** [comment, marked as answer]
Easy to accomplish - >

- Go to the repository you want to add and make sure it has an "Issues" section enabled.
- On the Community Exchange page, click on the "Add repository" button.
- Search for your repository by typing its name in the search bar.
- Once you find your repository, click on the "Add repository" button next to it.
- If the repository is successfully added, you should be able to see it on the Community Exchange page.
-
Done. :)

NOTE -> If the repository belongs to your organization, make sure you have the necessary permissions to add it to the Community Exchange. Otherwise it will not happen for you.

**Reply by ArtyomVancyan** [reply]
You are right, I can enable the issues section for a fork. But let's focus on the original repository. So, how can I grant my organization permission to Community Exchange?

I hoped to manage that by going to:

Settings -> Applications -> Authorized OAuth Apps

then grant my organization permission. But I did not find a way to do it.

**Reply by jge162** [reply]
To grant your organization permission to Community Exchange, you will need to be an organization administrator. 

- Go to your organization's settings page on GitHub.
- Click on the "OAuth Apps" tab.
- Locate the Community Exchange app and click on "Authorize."
- Follow the prompt to grant authorization to Community Exchange.

**Reply by ArtyomVancyan** [reply]
Thanks for the reply.
![Screenshot from 2023-02-04 14-27-30](https://user-images.githubusercontent.com/44609997/216762015-741d22b3-af0f-4f39-bb16-71c8215ad163.png)
On the "OAuth Apps" tab, I did not find anything.
![Screenshot from 2023-02-04 14-28-01](https://user-images.githubusercontent.com/44609997/216762019-17b8d6a6-40d2-4f32-91d7-581eee278d2d.png)

**Reply by jge162** [reply]
If you don't see any OAuth apps listed under your organization's settings, In this case, you may need to reach out to the Community Exchange support team and ask them to grant authorization to your organization. you need to contact [Github Support](https://support.github.com/contact?tags=dotcom-direct%2Chubberfy_building-communities%2Crr-general-technical)

**Reply by ArtyomVancyan** [reply]
I wrote them yesterday, opened a ticket and was waiting for the answer. I thought I'll get the answer more quickly. Although, thank you very much for your time, and wish you a good day.

**Reply by ATREAY** [reply]
Hey @ArtyomVancyan, did you hear back from GitHub? I'm having the same issue.

**Reply by ArtyomVancyan** [reply]
Nope
